lib/3scale_toolbox/commands/product_command/import_command.rb

```diff
@@ -0,0 +1,125 @@
+module ThreeScaleToolbox
+  module Commands
+    module ProductCommand
+      class ImportSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+        include ThreeScaleToolbox::ResourceReader
+
+        def self.command
+          Cri::Command.define do
+            name        'import'
+            usage       'import [opts] <remote>'
+            summary     'Import product from serialized format'
+            description 'This command deserializes one product and associated backends'
+
+            option      :f, :file, 'Read from file instead of stdin', argument: :required
+            ThreeScaleToolbox::CLI.output_flag(self)
+            param       :remote
+
+            runner ImportSubcommand
+          end
+        end
+
+        def run
+          validate_artifacts_resource!
+
+          product_list.each do |product|
+            context = {
+                target_remote: remote,
+                source_remote: crd_remote,
+                source_service_ref: product.system_name,
+                logger: Logger.new('/dev/null')
```

Comment:
Is this intended to be /dev/null?

Reply:
Yes, for backwards compatibility, the `product copy` command still prints messages. But for the new command, we do not want those messages to be printed. Instead, the new command generates in stdout a report in json format with structured info.

Reply:
:+1: